### DISCUSSION TITLE
What is the origin of CSS ?

### DISCUSSION THREAD

**Discussion by montasim** [STRINGLABITSOLUTIONS/Bhojon#214] [Q&A] [answered]
In short.

**Comment by ankit-akash** [comment, marked as answer]
CSS stands for Cascading Style Sheet. It was Lie and Tim Berners-Lee at CERN were the founder of CSS in the 1990s but it was first proposed by Håkon Wium Lie on 10 October 1994.CSS1 was released in 1996.Since, style sheet languages like DSSSL and FOSI were not suitable as style-sheets were reqquired to come from different sources on the web.As HTML grew,it became common to use CSS and web developers started preferring CSS over other style sheet languages.

**Reply by montasim** [reply]
@ankit-akash congratulations. Now you have a galaxy brain badge.
